### DISCUSSION TITLE
【求解】Solidity入门 2.数值类型 定长字节数组

### DISCUSSION THREAD

**Discussion by Durian-boy** [AmazingAng/WTF-Solidity#405] [Q&A] [answered]
_byte32为0x4d696e69536f6c69646974790000000000000000000000000000000000000000
bytes1 public _byte = _byte32[0];
_byte变量存储_byte32的第一个字节，为0x4d
//以上摘自教程

疑问：有查到0x是16进制的前缀，代表后面的数值是16进制，那么这里的_byte是不是4d69？
0x之后的数值长度为64，按16进制2个数值占一个字节算，好像没有位置存储“0x”这部分了，请问0x是如何存储的？

**Comment by AmazingAng** [comment, marked as answer]
0x没有被存储，以16进制读取时候加上，方便显示格式。
